File: aoj/volume100/csharp/A10027.cs
using System;

class AOJ
{
	static void Main()
	{
		var scoreA = 0;
		var scoreB = 0;
		var noGame = int.Parse(Console.ReadLine());
		for (var i = 0; i < noGame; i++)
		{
			var input = Console.ReadLine().Split();
			var a = input[0];
			var b = input[1];

			switch (a.CompareTo(b))
			{
				case 0:
					scoreA += 1;
					scoreB += 1;
					break;
				case 1:
					scoreA += 3;
					break;
				case -1:
					scoreB += 3;
					break;
			}
		}

		Console.WriteLine("{0} {1}", scoreA, scoreB);
	}
}

File: aoj/volume100/nodejs/A10027.js
function main(lines) {
  var score_a = 0;
  var score_b = 0;

  var i = 0;
  var noGemes = parseInt(lines[i++], 10);
  for (var gi = 0; gi < noGemes; gi++) {
    var cards = lines[i++].split(/\s+/);
    if (cards[0] === cards[1]) {
      score_a += 1;
      score_b += 1;
    } else if (cards[0] > cards[1]) {
      score_a += 3;
    } else if (cards[0] < cards[1]) {
      score_b += 3;
    }
  }
  console.log(score_a, score_b);
}

var chunks = [];
process.stdin.on('data', function(chunk) {
  chunks.push(chunk)
});
process.stdin.on('end', function() {
  var lines = chunks.join('').split('\n');
  main(lines);
});
